### PR TITLE
Search bar focus node feature is now working

### PR DESCRIPTION
**Pull request by SirenaYu**
-created JSON adjacency list in backend, specifically in views.py
-update_focused_node function now displays only the focused node and its associated nodes when a name is searched

### CODE REVIEW

**Comment by mscuthbert** [comment]
Just noticed that this has sat for 7 days.  If R or S don't have time to review, I will.

**Comment by srisi** [comment]
> Just noticed that this has sat for 7 days. If R or S don't have time to review, I will.

My bad. Will review now.

**Comment by srisi** [comment]
@SirenaYu: Is this ready to be reviewed again? And can you resolve the merge conflicts with master?
Thanks!

File: frontend/src/main/graph.js [hunk 1]
```diff
@@ -8,6 +8,19 @@ import * as d3 from 'd3';
 // so turning off standard eslint indent rules just for this file
 /* eslint indent: 0 */
 
+export function get_adj_list(data){
```
[the remark this answers is not in the document]
Comment (SirenaYu) [reply frontend/src/main/graph.js]: As for now, yes, because create_graph still uses this function. I believe @angcast is fixing it tho!

**Comment by SirenaYu** [comment]
I merged search_bar_unfocusing into search bar so the other PR (#45) should also be included in this now. Please check if the changes works!

**Pull request merged**